lib/components/tag-chip/index.tsx

```diff
@@ -1,29 +1,41 @@
 import React, { FunctionComponent } from 'react';
 import SmallCrossIcon from '../../icons/cross-small';
+import RevisionsIconSmall from '../../icons/revisions-small';
 import classNames from 'classnames';
 
 import type * as T from '../../types';
 
 type OwnProps = {
   onSelect?: (event: React.MouseEvent<HTMLDivElement>) => any;
-  selected: boolean;
+  selected?: boolean;
+  interactive?: boolean;
+  deleted?: boolean;
   tagName: T.TagName | undefined;
 };
 
 const TagChip: FunctionComponent<OwnProps> = ({
   onSelect,
-  selected,
+  selected = false,
+  interactive = true,
+  deleted = false,
   tagName,
 }) => (
   <div
-    className={classNames('tag-chip', { selected })}
+    className={classNames('tag-chip', { selected, interactive, deleted })}
```

Comment:
is it truly "deleted" or just something like transitory or temporary or not current? I'm thinking that deleted might suggest that the tag itself was deleted vs. removed from this particular note. maybe that's not the case. tags are confusing.

Reply:
The deleted state is used for those tags that are assigned to a note but don't exist in the global state (most likely because they were deleted). This case would only happen when displaying revisions of a note, they might have tags that were deleted.

Reply:
oh interesting! I didn't understand that from the code.

Reply:
Perhaps I should use different naming if it's a bit misleading 🤔 .

Regarding my previous comment, you can check the code related to how deleted tags are calculated [here](https://github.com/Automattic/simplenote-electron/pull/2817/files#diff-662df48571826155fa958190db3aa7146f9dd332ca44151a8ea4f1ad554a7258R63-R71).